CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute
+
+cert-manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into cert-manager easier.
+
+# Email and chat
+
+- Email: [giantswarm](https://groups.google.com/forum/#!forum/giantswarm)
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) IRC channel on freenode.org
```

Comment:
Where did you copy this file from?

I guess nobody is lurking in the freenode channel anymore and the google group also looks dead.

Reply:
A while back I checked a number of repos for contributing guidelines, one of them was [openstack-app-collection](https://github.com/giantswarm/openstack-app-collection/blob/master/CONTRIBUTING.md). I don't remember all others but there were a few.

Seems like this part is outdated and nobody updated it. Any idea where I could find infos about which contact info to replace that with?

Reply:
The file is probably outdated in most of the repositories (wild guess).

You could think about automatically keeping this file updates through giantswarm/github. Content-wise I have no idea. GitHub is probably the easiest for us, but there are maybe people that don't want to create an account.

Reply:
Well... I asked myself the same: Who's using IRC? :D Thought we might have some kind of Slack integration...